.github/workflows/codeflash.yml

```diff
@@ -13,7 +13,7 @@ concurrency:
 jobs:
   optimize:
     name: Optimize new Python code in this PR
-    if: ${{ github.actor != 'codeflash-ai[bot]' }}
+    if: ${{ github.actor == 'codeflash-ai[bot]' }}
```

Comment:
I clearly didn't know what I was doing here. However, this workflow failed completely, because it did not have access to relevant credentials stored within GitHub secrets.

Reply:
Aye. I've removed fea3a50 again.

Reply:
Now, receiving this:
```python
AssertionError: Codeflash API key not found. When running in a Github Actions Context, provide the 'CODEFLASH_API_KEY' environment variable as a secret.
```

![image](https://github.com/user-attachments/assets/c0cc0537-a241-461a-8d08-9b9104de6d16)

-- https://github.com/langflow-ai/langflow/actions/runs/13186924013/job/36811137830?pr=6011#step:5:37